### ISSUE TITLE
[BUG] Issue with assigning categories to some items.

### ISSUE THREAD

**Issue by RCIRacing** [arkayenro/arkinventory#1577]
**What version number you are using?**
[3.09.67] (https://www.curseforge.com/wow/addons/ark-inventory/files/3817945)

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Describe the bug**
Realistically, just see attached screenshot. Happens to quite a few items.

[WoWScrnShot_061022_133140](https://user-images.githubusercontent.com/69708288/173146735-7804d0e1-6da3-40bc-b956-2ba1febf8b62.jpg)

**To Reproduce**
N/A

**Screenshots**
N/A

**Additional context**
N/A

**Comment by arkayenro** [comment]
**locked** means youre using one of the system blueprint options that cannot be changed.
you need to change to, or create, one of your own blueprint in your profile

see https://github.com/arkayenro/arkinventory/wiki/UserGuide_Blueprints

**Comment by RCIRacing** [comment]
Okay, I forgot about that, this reminded me. That's the other issue I was having -- I had designs / category sets created (as I've been using AI since...Legion probably I'd imagine?), but they wouldn't work with 9.x. I had deleted them to recreate and every time I'd create one, the menu would go completely blank and do nothing.

That said, I just backed/wiped my variables from AI just to try, and that seems to solve that issue as well. Something it didn't like in the old 8.x settings I think. Blizz punishing me for not playing for a year!
